src/core/src/lib.rs

```diff
@@ -145,25 +145,38 @@ impl<R: Resources> ShaderSet<R> {
     }
 }
 
-//TODO: use the appropriate units for max vertex count, etc
 /// Features that the device supports.
-#[allow(missing_docs)] // pretty self-explanatory fields!
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
 #[cfg_attr(feature = "serialize", derive(Serialize, Deserialize))]
 pub struct Capabilities {
-    pub max_vertex_count: usize,
-    pub max_index_count: usize,
+    /// Recommended maximum number of vertices per draw call.
+    /// If there is no recommendation, the value is `None`.
+    pub max_vertex_count: Option<VertexCount>,
```

Comment:
Ah, I see that D3D11 just have those counts as 2^32. How about we remove them entirely then?